src/timeline/Timeline.tsx

```diff
@@ -13,6 +13,13 @@ const LEFT_MARGIN = 60 - 1;
 const {width: dimensionWidth} = Dimensions.get('window');
 
 export interface TimelineProps {
+  /**
+   * The date of this timeline instance
+   */
+  date?: string;
```

Comment:
Is there a required format to the date string? If so you should add it to the description

Reply:
Fixed